commands/corona/index.js

```diff
@@ -264,16 +264,20 @@ module.exports = {
 			word: (allRecoveries === 1) ? "recovery" : "recoveries"
 		};
 	
+		const active = group(allCases - (allDeaths ?? 0) - (allRecoveries ?? 0));		
```

Comment:
The `group` method returns a string, and as such the per-million calculation will not work. Skip the `group` call first, and calculate the per-million number inside of the `active !== null` branch, and then use `group` as necessary.